tensorboard/plugins/mesh/tf_mesh_dashboard/mesh-viewer.ts

```diff
@@ -50,6 +50,8 @@ export class MeshViewer extends THREE.EventDispatcher {
   private _animationFrameIndex?: number;
 
   private _camera?: THREE.PerspectiveCamera;
+  private _init_camera_position?: THREE.Vector3;
+  private _init_camera_look_at?: THREE.Vector3;
```

Comment:
JavaScript/TypeScript use camelCase instead of snake_case. Here and below. Please fix.